desktop_version/src/Script.cpp

```diff
@@ -798,6 +798,16 @@ void scriptclass::run(void)
                     graphics.textboxcentery();
                 }
 
+                if (map.custommode)
+                {
+                    int flags = PR_FONT_IDX(font::font_idx_level, cl.rtl);
```

Comment:
Print flags are always `uint32_t`, so:

```suggestion
                    uint32_t flags = PR_FONT_IDX(font::font_idx_level, cl.rtl);
```